cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/query/AbstractIndexTestBase.java

```diff
@@ -43,18 +43,16 @@ public void setUp() throws Exception {
         assertThat(ds, is(notNullValue()));
         im = new IndexManager(ds);
         assertThat(im, is(notNullValue()));
-        db = TestUtils.getDatabaseConnectionToExistingDb(im.getDatabase());
-        assertThat(db, is(notNullValue()));
-        assertThat(im.getQueue(), is(notNullValue()));
+        assertThat(im.getDatabaseQueue(), is(notNullValue()));
```

Comment:
The `SqlDatabase db` field can be removed now that the value write for it has been deleted.